cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -35,7 +35,7 @@ desiredState:
                     "-domain={{ pillar['dns_domain'] }}",
             ]
           - name: skydns
-            image: kubernetes/skydns:2014-12-23-001
+            image: skynetservices/skydns:latest
```

Comment:
And, unless they have fixed the SkyDNS container to not be 600 MB, we should keep using our own build.  @bketelsen have you guys "fixed" that yet?